galaxy_ng/app/settings.py

```diff
@@ -341,7 +341,9 @@
     'ldap': [
         "galaxy_ng.app.auth.ldap.GalaxyLDAPBackend",
         "django.contrib.auth.backends.ModelBackend",
-        "pulpcore.backends.ObjectRolePermissionBackend"
+        "pulpcore.backends.ObjectRolePermissionBackend",
+        "galaxy_ng.app.auth.ldap.PrefixedLDAPBackend",
```

Comment:
Isnt this overriding the `GalaxyLDAPBackend` added on the top? does it mean the system will attempt LDAP authentication twice?